### ISSUE TITLE
[Accessibility]A11y_VSCode_KeyboardShortcuts_Keyboard: Loss of focus occurred after navigation of "Change Keybinding" button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52031]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
 Keyboard,MAS2.4.7

**Repro Steps:**
1. Navigate to Settings icon in Activity Bar->Press Enter->Select "Keyboard Shortcuts" from dropdown.
2.Navigate to "Keybindings.json" link and using tab navigate to "Change Keybinding" button.
3.Observed that after "Change keybinding" button keyboard focus is loss.

**Actual:**
While navigating through the "tab" key the keyboard focus loss is occured,
after navigating the "change keybinding" button.

**Expected:**
Keyboard focus should be visible on each actionable control.
Here,keyboard focus should be visible after clicking on "Change Keybinding" button.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/fixit-details.aspx?fixitid=317](url)

**User Impact:**
User will not know where the keyboard focus is.

**MAS Reference:**
MAS2.4.7- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_KeyboardShortcuts_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2106001/Accessibility.A11y_VSCode_KeyboardShortcuts_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by AccessibilityTestingTeam-TCS** [comment]
Here while using tab, on each tab action keyboard focus is visible on each actionable control.When user reached on "Change Keybinding" button and then clicked tab key then keyboard focus is loss.User can not identify where the keyboard focus goes after "Change Keybinding" button.
Issue is also mentioned in attached ppt.
